Src/Utils/children.py

```
@@ -0,0 +1,26 @@
+from collections import deque
+
+import dearpygui.dearpygui as dpg
+
+
+def get_children(item: str | int, depth: int = None, slot: int = 1) -> set[int | str]:
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)